doc/Standaard-inrichting-GitHub-GitLab.md

```diff
@@ -58,11 +65,14 @@ De licentie tekst is voor elke VNG Realisatie GitHub repository gelijk en wordt
 2. Kopiëer de inhoud van een LICENCE.md bestand uit een andere VNG-Realisatie repository, plaats deze in het nieuwe bestand en commit het bestand direct
 naar de 'main' branch.
 
+VRAAG: Hebben we voor alles dezelfde licentie? Wie is verantwoordelijk voor de licentie? De beheerders van de github omgeving of de PO van de standaard?
 
 ### Initiële structuur
 
 We streven er binnen de VNG-Realisatie GitHub repositories naar overal een gelijke structuur te hanteren. Het gaat daarbij niet alleen over de folder structuur maar ook over de branch structuur. E.e.a. is nog wel afhankelijk van de functie van de repository. Zo wordt niet elke repository gebruikt t.b.v. een API specificatie en in dat geval is de folder 'specificatie' natuurlijk niet nodig. Wordt de repository echter wel gebruikt met als doel om een API specificatie op te leveren dan staat die specificatie wel in de folder 'specificatie'. 
 
+VRAAG: wat staat er precies in de folder 'specificatie'? Ik zou hier de foldernaam 'standaard' voor gebruiken. Dan kunnen daaronder folders voor documentatie, tutorials, yaml etc. komen. Bevat de repository voor een API standaard ook de code van de referentie implementatie? 
```

Comment:
In ZGW wordt hier de naam 'api-specificatie' voor gebruikt. Jullie hebben die folder weer verder onderverdeeld voor de diverse API's. Bij HC hebben we in de folder 'specificatie' veelal alleen de folder 'genereervariant' staan.

Ik denk dat het niet haalbaar is om bij ZGW of bij HC de naam van deze folder aan te passen, dat gaat toch niet gebeuren. Waar we wel naar zouden kunnen streven is om in de toekomstige repo's wel een standaard naam te hanteren. Bij regels bij Activiteiten heb ik de HC wijze gebruikt.

Waar we misschien ook nog iets over moeten zeggen is over de opdeling van de API specificaties over repositories. Bij ZGW zie je dat alle API specificaties in 1 repo staan. Bij HC heeft elke API zijn eigen repo.